### ISSUE TITLE
what should be changed if there is only two classes

### ISSUE THREAD

**Issue by YesDargo** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#25]
Hi, I am new to the ML/CV, now I am doing image segmentation for skin lesion. I only need to separate lesion area from background (two classes)

When I configure the main.py and run train, the `loss` remains in 1.231 after a few steps. Then I run test, the `pixel accuracy` keeps 1.00 and `mean IoU` keeps 0.5. Did you encounter the same problem?

I found in Dr.Sleep's note that when load checkpoint for different class number (not 21), the `--not-restore-last` should be passed. Have you also implement this?

**Comment by John1231983** [comment]
In my opinion, just change the class number to 2. Increase weight to 1e-3 and sure that you have enough training data. Otherwise, you can retrain from resnet101

**Comment by YesDargo** [comment]
Thanks for reminding!
The initial setting was to load the checkpoint from `deeplab_resnet_init.cpkt`.  Now I change to `deeplab_resnet.cpkt`, and the loss starts to drop to 1.189 from 1.231.
